lms/djangoapps/grades/signals/handlers.py

```diff
@@ -123,7 +125,23 @@ def score_published_handler(sender, block, user, raw_earned, raw_possible, only_
                 )
 
     if update_score:
+        course = get_course(block.course_id)
+        original_course_grade = CourseGradeFactory().create(user, course)
+
+        # Set the problem score in CSM.
         score_modified_time = set_score(user.id, block.location, raw_earned, raw_possible)
+
+        course_overview = CourseOverview.objects.filter(id=block.course_id).first()
+        course_grade = CourseGradeFactory().create(user, course)
+
+        grade_changed = original_course_grade.percent != course_grade.percent
```

Comment:
how about getting rid of the `grade_changed` var? it's used only once

Reply:
I think it makes code more readable, doesn't it?